Project.toml

```diff
@@ -5,10 +5,12 @@ version = "0.2.2"
 
 [deps]
 Catlab = "134e5e36-593f-5add-ad60-77f754baafbe"
+ComponentArrays = "b0b7db55-cfe3-40fc-9ded-d10e2dbeff66"
 Compose = "a81c6b42-2e10-5240-aca2-a61377ecd94b"
 LabelledArrays = "2ee39098-c373-598a-b85f-a56591580800"
```

Comment:
I think we can remove LabelledArrays at this point. This will be a good check that we got all the occurrences and don't need it anymore.